### DISCUSSION TITLE
[Question] Why running a model straight from FG results in prop not found errors?

### DISCUSSION THREAD

**Discussion by darssy** [JSBSim-Team/jsbsim#441] [Q&A]
Hello everyone,

I am (very) new to JSBsim and I thought I should try it as it _might_ be helpful in my own simulation. So I downloaded the latest installer (Release v1.1.6) and I tried to run the example:

` ./JSBSim.exe --script=scripts/c1721.xml`

Cool I thought to myself now let's try another script. I picked `737_cruise.xml`. Excellent. Then I tried to add aircraft straight from flight gear. I tried [this one](https://github.com/YV3399/737-800YV) or even the [official 738](https://sourceforge.net/p/flightgear/fgaddon/HEAD/tree/trunk/Aircraft/737-800/) and I was receiving several errors about the properties. For example ` The property /sim/model/pushback/position-norm does not exist`.

Ok I thought, I don't need pushback for cruise, let's remove the reference to pushback. But then there were other errors following so I started removing any offending part of the script ending up with several modifications. Even then I received a `FATAL ERROR: JSBSim terminated with an exception. The message was: Trim Failed`.

Removing parts of the configuration of an aircraft already looks fishy to me and is an indication that I am doing something wrong. So here is my question: am I missing something, or the FG aircraft aren't meant to run by JSBSim out of the box?

Thanks in advance.

**Comment by bcoconni** [comment]
You may already know that but FlightGear is a full featured flight simulator whereas JSBSim is an FDM, and to some extent, FlightGear is a superset of JSBSim. As a consequence, there are much more properties such as those you mentioned which exist in FlightGear than there are in JSBSim. So it is not unexpected that a JSBSim model that has been specifically designed for the purpose of being used in FlightGear makes reference to properties that are specific to FlightGear.

This is most unfortunate but some authors of FlightGear aircraft (if not most them) do not check if their FDM successfully runs in JSBSim standalone. It is likely that these authors either do not know that JSBSim is an independent project or they may have no interest in using JSBSim as an independent program. So the short story is that you are not doing anything wrong, just that some FDM from FlightGear indeed need to be cleaned up before being used in JSBSim standalone.

**Reply by darssy** [reply]
Hello @bcoconni and thanks for getting back to me with an explanation. I do realize of course that FG is a full featured simulator. I also had in mind that FG uses _other_ FDMs like YASIM for example. As such I would expect that the FDM (any FDM) would be fully segregated from the rest of the simulation and a model description would only contain FDM related properties and not properties that would make the FDM explode ☹

In the meantime I found this one as well in FG forums which confirms what you already wrote. https://forum.flightgear.org/viewtopic.php?f=49&t=29803

Thanks anyway 🙂

PS: sorry for hijacking the issues system to ask a question, I didn't notice there was a dedicated discussion tab. :(
